docs/install/configuration.md

```diff
@@ -60,8 +60,12 @@ See [here](./email_providers.md) for example configuration with common email pro
 By default, the platform will send anonymous usage information back to us at FlowForge Inc.
 This can be disabled via the Admin Settings in the UI, or turned off in the configuration file.
 
-For more information about this feature, see [here](../admin/telemetry.md)
+Additionally, you can configure your own instance of FlowForge to report back to you on how users are using FlowForge. FlowForge is designed to work with [Plausible](https://plausible.io/). You can setup your own account, and pass the releavnt domain to the `yml` in the telemetry configuration
+
+For more information about this feature, see [here](/docs/admin/telemetry.md)
 
 Option        | Description
 --------------|------------
 `telemetry.enabled` | Enables the anonymous usage telemetry of the platform. Default: `true`
+`telemetry.plausible.domain` | The `data-domain` of your site (see [Plausible docs](https://plausible.io/docs/plausible-script)). Default: `null`
```

Comment:
Can we take the domain value from the `base_url` value to save the user having to enter it twice?

Reply:
yes, although not sure I see why they're entering twice? Isn't it just the once in this config?

Reply:
@hardillb possibly - it depends what using `localhost:3000` for your base_url would do with Plausible - given they use the page referrer to identify where the data should be sent.

It may be we need to be able to override it

Reply:
@joepavitt I meant that the user already has to enter a `base_url` value that is the main entry point for the site, we extract the hostname/domain from that rather than having to type it in a second time to setup to Plausible

Reply:
@hardillb that seems sensible. I do want to differentiate in the yml though the Plausible being enabled, separate from the general telemetry, so what would be the alternative config structure?

Reply:
```yaml
telemetry:
  enabled: true
  frontend:
    plausible:
      enabled: true
      domain: <data-domain-here>
      extension: local
```

So that `plausible.domain` can be optional to override the one extracted from the `base_url` value (say the site gets moved, but wants to keep the data history)

Reply:
OK, that makes sense. I was just trying to reduce duplication in the config file.